lib/crud.js

```diff
@@ -0,0 +1,34 @@
+export class CRUD {
```

Comment:
I think its clever idea to regroup all queries into a share file. I would prefer the name `queries.js` so its more generic and use export function instead of and objet like : 
```js
export  async findSensors() {
    await fetch(this.url, {
      next: { revalidate: 10000 },
    })
  }
```
And make it work with an `useQuery` from [react-query](https://tanstack.com/query/latest)